strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenSearchControllerTest.java

```diff
@@ -31,99 +32,84 @@
 public class MavenSearchControllerTest
```

Comment:
same changes was done within [#1159](https://github.com/strongbox/strongbox/pull/1159) , so I suggest to wait it instead